src/thor/bidirectional_astar.cc

```diff
@@ -571,7 +573,7 @@ BidirectionalAStar::GetBestPath(valhalla::Location& origin,
 
         // Terminate if the cost threshold has been exceeded.
         if (fwd_pred.sortcost() + cost_diff_ > threshold_) {
-          return FormPath(graphreader, options, origin, destination);
+          return FormPath(graphreader, options, origin, destination, forward_time_info, invariant);
```

Comment:
not sure if it's fully correct to consider only forward time and ignore reverse time:
- on the one hand we adjust times here https://github.com/valhalla/valhalla/blob/master/src/thor/route_action.cc#L409 and here https://github.com/valhalla/valhalla/blob/master/src/thor/route_action.cc#L512 ;
- but on the other hand we potentially can miss time restrictions in case we have predefined arrival time.